src/App.jsx

```diff
@@ -14,45 +16,91 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+function getPreparedGoods(goods, { sortField, isReversed }) {
+  let allGoods;
+
+  switch (sortField) {
+    case 'alphabet':
+      allGoods = [...goods].sort();
+      break;
+    case 'length':
+      allGoods = [...goods].sort((good1, good2) => good1.length - good2.length);
+      break;
+    case '':
+    default:
+      allGoods = [...goods];
+      break;
+  }
+
+  if (isReversed) {
+    allGoods.reverse();
+  }
+
+  return allGoods;
+}
+
+export const App = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
It'd be nice to create variables for different sort values:
`const SORT_ALPHABETICALLY = 'alphabet';`
And then use it everywhere.